gcloud/bigtable/happybase/table.py

```diff
@@ -23,6 +23,7 @@
 from gcloud.bigtable.column_family import GCRuleIntersection
 from gcloud.bigtable.column_family import MaxAgeGCRule
 from gcloud.bigtable.column_family import MaxVersionsGCRule
+from gcloud.bigtable.happybase.batch import _WAL_SENTINEL
```

Comment:
Ditto. I almost ripped it out when porting from `gcloud-python-bigtable` but then I saw why it's done: the default `wal` is passed down into `batch` helpers.

I plan on ripping this out and using `**kwargs` with a warning as is done in the `Connection` constructor and as is done here in `Table.scan`.